profiles/README.md

```diff
@@ -10,13 +10,44 @@ Specifically, we include:
 2. Processed data for each Cell Painting plate (for several "data levels")
 3. Instructions on how to reproduce the profiling pipeline
 
-## Workflow
+## Workflow 
 
-![Cytominer workflow](media/cytominer_workflow.png)
+![Cytominer workflow](media/cytominer_workflow_new.png)
+
+## Pycytominer
+[Pycytominer](https://github.com/cytomining/pycytominer) is a code base built by @gwaygenomics and @niranjchandrasekaran. 
```

Comment:
I guess you could say

```suggestion
[Pycytominer](https://github.com/cytomining/pycytominer) is a code base built by @gwaygenomics et al.
``` 

I don't think my contribution is significant enough to have my name appear here 🙂